doc/source/f2py/signature-file.rst

```diff
@@ -234,172 +241,188 @@ Other statements:
+        ``callstatement`` and ``callprotoargument`` can be used only if
+        they are valid for all entry routines, etc.
+
+F2PY statements
+^^^^^^^^^^^^^^^^
 
   In addition, F2PY introduces the following statements:
 
-  + ``threadsafe``
-      Use ``Py_BEGIN_ALLOW_THREADS .. Py_END_ALLOW_THREADS`` block
-      around the call to Fortran/C function.
-
-  + ``callstatement <C-expr|multi-line block>``
-      Replace F2PY generated call statement to Fortran/C function with
-      ``<C-expr|multi-line block>``. The wrapped Fortran/C function
-      is available as ``(*f2py_func)``. To raise an exception, set
-      ``f2py_success = 0`` in ``<C-expr|multi-line block>``.
-
-  + ``callprotoargument <C-typespecs>``
-      When ``callstatement`` statement is used then F2PY may not
-      generate proper prototypes for Fortran/C functions (because
-      ``<C-expr>`` may contain any function calls and F2PY has no way
-      to determine what should be the proper prototype). With this
-      statement you can explicitly specify the arguments of the
-      corresponding prototype::
-
-        extern <return type> FUNC_F(<routine name>,<ROUTINE NAME>)(<callprotoargument>);
-
-  + ``fortranname [<actual Fortran/C routine name>]``
-      You can use arbitrary ``<routine name>`` for a given Fortran/C
-      function. Then you have to specify
-      ``<actual Fortran/C routine name>`` with this statement.
-
-      If ``fortranname`` statement is used without
-      ``<actual Fortran/C routine name>`` then a dummy wrapper is
-      generated.
-
-  + ``usercode <multi-line block>``
-      When used inside ``python module`` block, then given C code
-      will be inserted to generated C/API source just before
-      wrapper function definitions. Here you can define arbitrary
-      C functions to be used in initialization of optional arguments,
-      for example. If ``usercode`` is used twice inside ``python
-      module`` block then the second multiline block is inserted
-      after the definition of external routines.
-
-      When used inside ``<routine signature>``, then given C code will
-      be inserted to the corresponding wrapper function just after
-      declaring variables but before any C statements. So, ``usercode``
-      follow-up can contain both declarations and C statements.
-
-      When used inside the first ``interface`` block, then given C
-      code will be inserted at the end of the initialization
-      function of the extension module. Here you can modify extension
-      modules dictionary. For example, for defining additional
-      variables etc.
-
-  + ``pymethoddef <multiline block>``
-      Multiline block will be inserted to the definition of
-      module methods ``PyMethodDef``-array. It must be a
-      comma-separated list of C arrays (see `Extending and Embedding`__
-      Python documentation for details).
-      ``pymethoddef`` statement can be used only inside
-      ``python module`` block.
+``threadsafe``
```

Comment:
Maybe turn this into a definition list: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists

Reply:
I believe it renders as one...

![image](https://user-images.githubusercontent.com/4336207/134054227-c4e8be17-4f23-464c-bca1-9066c75bf582.png)